suite-common/graph/src/graphDataFetching.ts

```diff
@@ -409,6 +409,13 @@ export const getMultipleAccountBalanceHistoryWithFiat = async ({
             findOldestBalanceMovementTimestamp,
             fromUnixTime,
         );
+
+        // if there were no balance movements at all,
+        // findOldestBalanceMovementTimestamp resulted with start date being the same as end date
+        // use 1 year into past instead, otherwise getTimestampsInTimeFrame will throw range error
+        if (startOfTimeFrameDate.getTime() === endOfTimeFrameDate.getTime()) {
```

Comment:
I am not sure if this si best fix. I think it may be better to just return early here since if there is no movement we know that graph will just zero all the time. I think `return []` or return `[{ date: 1, value: 0 }, {date: 2, value: 0}]` should work and it will be faster because otherwise we will be fetching timestamps for that whole year.

Reply:
So in such case it should probably check for every time if there are at least some movements of any coin to return early for each option 1d/1w/1m/...

I was thinking about this, but it felt suboptimal for such edge case. And this is the same behaviour there is for 1y option.

Reply:
Ok. I'll modify the behaviour for empty history.

Reply:
I've added check for empty account in 2091d876ab and kept the timeframe update to better UX when touching the graph.

Reply:
Sorry but it seems unecessary complicated now, I need to take a proper look and think about it.

Reply:
I still think that best solution is return early if  `startOfTimeFrameDate === endOfTimeFrameDate` because that's easy and will fix the issue. Don't care about the other timeframes it's not worth added complexity.

Reply:
simplified [49b17fd](https://github.com/trezor/trezor-suite/pull/14506/commits/49b17fd0b5ef5065cce409f9e1a7d69e19f0f1d9)